plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -825,7 +824,7 @@ public DeltaLakeOutputTableHandle beginCreateTable(ConnectorSession session, Con
         Path targetPath = new Path(location);
```

Comment:
Did you omit this occurence of `org.apache.hadoop.fs.Path` ?

Reply:
Those changes are in follow up PR: https://github.com/trinodb/trino/pull/16469